graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -309,6 +310,9 @@ public Stream match(Message message) {
                     LOG.debug("Error matching stream rule <" + rule.getType() + "/" + rule.getValue() + ">: " + e.getMessage(), e);
                 }
                 streamMetrics.markExceptionMeter(streamId);
+                final String error = f("Error matching stream rule <%s> (%s)", streamRuleId, rule.getDescription());
```

Comment:
could we add the type and value as well? (see above debug message)
Finding the rule by ID might not be so easy in the UI. 
And also the streamId that this rule is referenced on might be useful to find it.